docs/_template.html

```diff
@@ -3,32 +3,31 @@
 
 <head>
   <meta charset="utf-8">
-  <title>{{fsdocs-page-title}}</title>
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="author" content="{{fsdocs-authors}}">
+  <title>{{fsdocs-page-title}}</title>
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Reem+Kufi:wght@400;500;700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
-  <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
-  <script type="text/javascript" async
-    src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=TeX-MML-AM_CHTML"></script>
   <link rel="shortcut icon" type="image/x-icon" href="{{root}}images/favicon.ico">
   <link type="text/css" rel="stylesheet" href="{{root}}content/fsdocs-custom.css" />
   <link type="text/css" rel="stylesheet" href="{{root}}content/fsdocs-default.css" />
   <script type="text/javascript" src="{{root}}content/fsdocs-tips.js"></script>
   <!-- BEGIN SEARCH BOX: this adds support for the search box -->
+  <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
   <link rel="stylesheet"
     href="https://cdnjs.cloudflare.com/ajax/libs/JavaScript-autoComplete/1.0.4/auto-complete.css" />
   <!-- END SEARCH BOX: this adds support for the search box -->
   {{fsdocs-watch-script}}
 </head>
 
 <body>
-  <main>
-    <div id="sidebar-wrapper" class="bg-primary h-100" >
+  <main >
+    <div id="sidebar-wrapper" class="bg-primary h-100 collapse" >
+      <div id="sidebar-logo" class="d-flex justify-content-center p-5">
       <a href="{{fsdocs-logo-link}}"><img id="fsdocs-logo" src="{{root}}/{{fsdocs-logo-src}}" /></a>
```

Comment:
This links back to the homepage (onepager), maybe it should be `{{fsdocs-logo-link}}/docs` instead.